### ISSUE TITLE
How to change default scaling?

### ISSUE THREAD

**Issue by zilexa** [ONLYOFFICE/Docker-DocumentServer#665]
Running DocumentServer via docker, all works, but how can I change the interface scaling?
Everything looks pretty tiny. I can zoom into the document but the UI elements stay small.

I also use Desktopeditors and I am able to set 125% permanently.

**Comment by igwyd** [comment]
Hello @zilexa, sorry for the late reply. You can use this parameter https://api.onlyoffice.com/editors/config/editor/customization#zoom to constantly change the document scale.

**Comment by zilexa** [comment]
@igwyd no, that is _document zooming_, I am clearly asking about UI scaling. 
This is supported in Desktopeditors for example in the settings. 

Many people have HiDPI screens especially on laptops. Making DocumentServer very difficult to use.

**Comment by igwyd** [comment]
You can scale the interface in the browser or, for example, in the operating system and documentserver support this. Do you want to scale using the documentserver configuration separately from scaling in the browser or OS?

**Comment by igwyd** [comment]
@zilexa please describe your request in more detail, could you provide the resolution and monitor size, as well as a screenshot of what your interface looks like?

**Comment by zilexa** [comment]
@igwyd my description is spot on. 
Using the browser to zoom only affects the document itself. Just like the document zoom option bottom right only affects the document itself. 
Meanwhile, the UI elements stay tiny. 

Below I used CTRL + in Firefox 118 to increase page size. As you can see, only the document itself is now huge. The UI elements stay tiny on my 1920x1280 screen on 13.5" HP Spectre x360:

![Screenshot from 2023-10-19 16-08-06](https://github.com/ONLYOFFICE/Docker-DocumentServer/assets/3430004/7193664e-98d9-4595-bd2c-568915f43ed1)
This is unusable. It is actually much smaller than my Firefox GUI which scales nicely with the operating system display configuration. 

Compare this with OnlyOffice DesktopEditors, which has a dedicated option for scaling (not the document, but the GUI itself): 
![Screenshot from 2023-10-19 16-11-37](https://github.com/ONLYOFFICE/Docker-DocumentServer/assets/3430004/24d4730e-f3ea-4829-9b5b-01610d310906)

**Comment by igwyd** [comment]
If you increase the browser zoom(or OS zoom) before open the document and then open it you will see that the UI incresed too, we focus on zoom in the browser. 

![screen](https://github.com/ONLYOFFICE/Docker-DocumentServer/assets/67538082/666d619f-b0c8-4c78-9c65-141d57c5e35a)

I see you point, you want to set up interface scaling via configuration independently of the scaling of the system or browser. I asked the developers a question about it.

**Comment by zilexa** [comment]
Currently it does not work like it does on ANY other website. 
When you open a document in Onlyoffice and the UI is too small, you hit CTRL+ to make it bigger, like on any other site. Or use the browser menu. The bug/issue: the document itself increases. 
Indeed, if there is no document open, the UI will finally increase. But that makes no sense. Especially if you use other fileservers and open files directly in OnlyOffice, there is no way to make the UI bigger. 

Since there is already a UI element to zoom into the document itself, as you pointed out with the red circle, the browser zoom capabilities should zoom the entire web application, not just 1 item inside it. Just like it does, on every other website.

**Comment by igwyd** [comment]
>When you open a document in Onlyoffice and the UI is too small, you hit CTRL+ to make it bigger, like on any other site. Or use the browser menu. The bug/issue: the document itself increases.

Actually our UI can to make bigger with ```CTRL+```, you shuld to move the mouse cursor over the toolbar and then use the ```CTRL+```.

**Comment by zilexa** [comment]
A year later, the issue has been explained by me and others several times. Yet the responses are still not related to the actual issue.

**Comment by igwyd** [comment]
@nrvale0 i understand it and I tried to explain how it works for us. I checked on my test linux PC, all right.

https://github.com/user-attachments/assets/0ca7c7d1-cab2-4cce-a9bc-dd6ba108d989

@zilexa I thought we had figured everything out. As i write [before](https://github.com/ONLYOFFICE/Docker-DocumentServer/issues/665#issuecomment-1870995246) here the developer explained to me that we won’t be able to set this using configuration parameters because we are focusing on the browser scale.

**Comment by igwyd** [comment]
> I am running OnlyOffice via .desktop XDG launching

I think you have a other case, please crete a new issue and detailed fill out the form, we will check it.
